Panel/Knobs/Inputs.js

```diff
@@ -0,0 +1,16 @@
+const React = require('react');
+
+const StringInput = ({ control, props, updateProps }) => (
+    <div>
+        <strong>{control.name}:  </strong>
+        <input
+            value={props[control.name]}
+            onChange={
+                (e) => updateProps(e, control)
```

Comment:
Event should give back updated value, not event, also given the context of this component, it should just be`updateProp` this component shouldn't have any knowledge of other props
```suggestion
                (evt) => updateProp(evt.target.value)
```